src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -218,10 +219,17 @@
     <PlatformManifestFileEntry Include="icudt_EFIGS.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_optimal.dat" IsNative="true" />
     <PlatformManifestFileEntry Include="icudt_optimal_no_CJK.dat" IsNative="true" />
-    <PlatformManifestFileEntry Include="runtime.iffe.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="dotnet.d.ts" IsNative="true" />
-    <PlatformManifestFileEntry Include="library-dotnet.js" IsNative="true" />
-    <PlatformManifestFileEntry Include="pal_random.js" IsNative="true" />
+    <PlatformManifestFileEntry Include="package.json" IsNative="true" />
```

Comment:
So that you could use it to install it as npm package into your user code. Also it enables .d.ts type definitions if you do that. 
This is just draft right now as it doesn't solve AOT use case yet.